Adds websocket-client to dependencies

jcoc611 opened the pull request:
A compliant websocket client library for packages that want to stay up to date with a server state.

<!--
Your pull request will be reviewed automatically and by a human.

The manual review may take several days or weeks, depending on the reviewer's availability and workload.
If you haven't received a comment on your pull request and it wasn't merged either,
it just hasn't been reviewed yet.

---

Please ensure the automated reviews pass. 
Follow the instructions provided, if necessary.
You can speed up the process
by [running some tests locally](https://packagecontrol.io/docs/submitting_a_package#Step_7).

You can trigger @packagecontrol-bot to re-evaluate your pull request
by pushing a commit or closing and reopening your pull request.
Do **NOT** open a new pull request!

In general, make sure you:

 1. Used `"tags": true` and not `"branch": "master"` 
    (versioning docs: <https://packagecontrol.io/docs/submitting_a_package#Step_4>)
 2. Added a README to your repository so that users (and reviewers) 
    can understand what your package provides.
 
You should proceed with a short description of what the package does
and, in case one or multiple similar package already exists, 
why you believe it is different and needed
below this line. -->

Thom1729 commented:
If it isn't too much hassle, I think it would make more sense to factor out `six` as a separate dependency. I know there are plenty of packages that would use it.

jcoc611 commented:
@Thom1729 Absolutely, I've gone ahead and added the six package as `python-six`, and updated the repo for `websocket-client` by removing its copy of `six`.
